modules/Behavior.js

```diff
@@ -63,12 +63,13 @@
     /**
      * @method constructor
      * @override
+     * @param behaviorAttributes {Object} the initial value of the behavior's attributes.
      * @param behaviorOptions {Object}
      *   @param behaviorOptions.view {Backbone.View} that Behavior is attached to
      *   @param behaviorOptions.alias {Backbone.View} the alias for the behavior in this view.
      * @param [viewOptions] {Object} options passed to View's initialize
      */
-    constructor: function(behaviorOptions, viewOptions) {
+    constructor: function(behaviorAttributes, behaviorOptions, viewOptions) {
```

Comment:
this is not backwards compatible and I don't see a good way to make it backwards compatible w/o going back to what it was.